myscaffold-domain/src/main/java/com/example/domain/common/message/MessageKeysGen.java

```diff
@@ -0,0 +1,49 @@
+package com.example.domain.common.message;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+
+import java.io.*;
+import java.util.regex.Pattern;
+
+public class MessageKeysGen {
+    public static void main(String[] args) throws IOException {
+        // message properties file
+        InputStream inputStream = new FileInputStream("myscaffold-web/src/main/resources/i18n/application-messages.properties");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code contains a resource that might not be closed properly. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *inputStream*. The resource is referenced in statements at the following lines: 14, 30. The resource closure statement is at line: 45. There are other execution paths that do not contain closure statements, for example, when FileUtils.openOutputStream throws an exception. Either a) close *inputStream* in a try-finally block or b) close the resource by declaring *inputStream* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).